Difi.SikkerDigitalPost.Klient/Api/SikkerDigitalPostKlient.cs

```diff
@@ -107,12 +118,12 @@ public Transportkvittering Send(Forsendelse forsendelse, bool lagreDokumentpakke
                 throw new Exception("Sending av forsendelse feilet under validering. Feilmelding: " + e.GetBaseException(), e.InnerException);
             }
 
-            var soapContainer = new SoapContainer { Envelope = forretningsmeldingEnvelope, Action = "\"\"" };
+            var soapContainer = new SoapContainer(forretningsmeldingEnvelope);
             soapContainer.Vedlegg.Add(arkiv);
-            var meldingsformidlerRespons = SendSoapContainer(soapContainer);
+            var meldingsformidlerRespons = await SendSoapContainer(soapContainer);
```

Comment:
Hvis du uansett bare venter på å få logget resultatet kunne du laget en async logg metode og fjernet await(som kan forårsake deadlocks)